Microsoft.NET.Build.Containers/Layer.cs

```diff
@@ -114,9 +115,9 @@ private static void EnsureDirectoryEntries(TarWriter tar, HashSet<string> direct
         var pathBuilder = new StringBuilder();
         for (var i = 0; i < filePathSegments.Count - 1; i++)
         {
-            pathBuilder.Append($"{filePathSegments[i]}/");
+            pathBuilder.Append(CultureInfo.InvariantCulture, $"{filePathSegments[i]}/");
```

Comment:
Changes like these came in from code style analyzers. I could split them out if desired.